src/CommunityToolkit.Maui.Core/Interfaces/IToast.shared.cs

```diff
@@ -22,12 +22,12 @@ public interface IToast : IAlert
 public enum ToastDuration
 {
 	/// <summary>
-	/// Displays Toast for a short time
+	/// Displays Toast for a short time (~2 seconds).
```

Comment:
I would prefer not to use explicit value as it may be internally changed. On Android it is an enum Short and Long

Reply:
Sure, but from the consumer side... How do I know what is short and long? Maybe it's just me, but I would love to have some indication how long it is. And yes, we would have to update as values change, isn't that always the case with comments?

Reply:
Comments are usually the place developers (maybe that's only me) forget to update.

Reply:
You're absolutely right! And that will probably happen and someone will open a bug for it ;)

Thanks!